ChangeLog.md

```diff
@@ -10,6 +10,10 @@
 |`msrest` dep of generated code | `0.6.21`
 |`azure-mgmt-core` dep of generated code (If generating mgmt plane code) | `1.3.0`
 
+**New Features**
+
+- Support new flag `--security`/`--security-scopes`/`--security-header-name` and security definition in swagger #945
```

Comment:
@msyyc the point isn't to add new flags in this PR, we only want to add support for security definitions in swagger

Reply:
autorest.python rely on modelerfour and modelerfour suports those flags. So we could declare to support those flags although it actually is handled by modelerfour.